clients/python/inspr/client.py

```diff
@@ -0,0 +1,57 @@
+import os
+import json
+import sys
+from flask import Flask, request, url_for
+from http import HTTPStatus
+from .rest import *
+from types import FunctionType
+
+SIDECAR_READ_PORT = "INSPR_SCCLIENT_READ_PORT"
+SIDECAR_WRITE_PORT = "INSPR_LBSIDECAR_WRITE_PORT"
+
+class Client:
```

Comment:
I'm not sure how python works but if there are two different libraries that declared the `Client` class, does it generate conflict?

Reply:
Maybe it wouldn't but might be nice to have something more descriptive like `InsprClient`

Reply:
Well, this happens in all other languages. You can just do:
`import inspr.client as insprClient`